src/saveload/afterload.cpp

```diff
@@ -3111,6 +3111,10 @@ bool AfterLoadGame()
 		FOR_ALL_INDUSTRIES(ind) if (ind->neutral_station != NULL) ind->neutral_station->industry = ind;
 	}
 
+	for (TileIndex t = 0; t < map_size; t++) {
```

Comment:
Should we guard this by a "up to and including the current savegame version" check?

Also i think a comment might be appropriate why this is here, for a person in the far future.

Reply:
This needs to be backported in which case the savegame version check is pointless, as we don't want to bump 1.9's version and it's not necessary anyway.

So we can add a savegame guard for master now, and remove it in the backport, or keep it as is here and add a savegame guard later in master?

Reply:
The guard would just prevent the conversion from running when not necessary. On the other hand, the conversion doesn't hurt anything, at least not currently.

Reply:
Should we bump master's savegame version then? And ignore that part for the backport?